peps/pep-0768.rst

```diff
@@ -294,6 +294,24 @@ An example usage of the API would look like:
     except Exception as e:
         print(f"Failed to execute code: {e}")
 
+Configuration API
+-----------------
+
+To allow redistributors, system administrators, or users to disable this
+mechanism, several methods will be provided to control the behavior of the
+interpreter:
+
+A new environment ``PYTHON_DISABLE_REMOTE_DEBUGGING`` environment variable will
+be provided to control the behaviour at runtime. If set to any value, the
```

Comment:
Does the empty string count as a value?

Reply:
I would say yes. WDYT?

Reply:
I'm ok either way, but I think the PEP should say "(including an empty string)" explicitly. There's a lot of people who don't realize that `unset FOO` and `FOO=` do different things...